package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@hashicorp/mktg-global-styles": "^3.0.2",
     "@hashicorp/mktg-logos": "^1.0.2",
-    "@hashicorp/nextjs-scripts": "^17.0.0",
+    "@hashicorp/nextjs-scripts": "^19.0.3",
```

Comment:
Can we drop this entirely now?

Reply:
I was running into some issues dropping it completely 😢  [`45c092e` (#256)](https://github.com/hashicorp/react-components/pull/256/commits/45c092e8df48c0e4cd75f2b0183029efbdf05b45) I think we need to get a publish fired off here with all the package deps pointing at the new packages, then that should let us fully drop the scripts dependency.